src/components/user/dto/list-users.dto.ts

```diff
@@ -22,6 +23,9 @@ export abstract class UserFilters {
   @OptionalField()
   readonly title?: string;
 
+  @OptionalField()
```

Comment:
Enum types cannot be inferred by TS, so they have to be explicitly declared. Currently it is inferred to be a `String`.
```suggestion
  @OptionalField(() => UserStatus)
```